docs/messaging/messaging-spans.md

```diff
@@ -329,7 +333,14 @@ under the namespace `messaging.destination_publish.*`
 the broker does not have such notion, the original destination name SHOULD uniquely identify the broker.
 <!-- endsemconv -->
 
-The *receive* span is used to track the time used for receiving the message(s), whereas the *process* span(s) track the time for processing the message(s).
+"Receive" spans SHOULD be created for operations of passing messages to the application when those operations are initiated by the application code.
+
+"Receive" spans MUST NOT be created for messages which are not forwarded to the caller, but are pre-fetched or cached by messaging libraries or SDKs.
```

Comment:
```suggestion
"Receive" spans MUST NOT be created for messages until they are forwarded to the caller, but are pre-fetched or cached by messaging libraries or SDKs.
```

otherwise it's possible to interpret it as that spans should never be created